log10/feedback/autofeedback.py

```diff
@@ -95,13 +101,47 @@ def predict(self, text: str = None, completion_id: str = None) -> str:
         return ret
 
 
+def get_autofeedback(completion_id: str) -> httpx.Response:
+    query = """
+    query OrganizationCompletion($orgId: String!, $id: String!) {
+        organization(id: $orgId) {
+            completion(id: $id) {
+                id
+                autoFeedback {
+                    id
+                    status
+                    jsonValues
+                    comment
+                }
+            }
+        }
+    }
+    """
+
+    variables = {"orgId": _log10_config.org_id, "id": completion_id}
+
+    response = _try_post_graphql_request(query, variables)
+
+    if response is None:
+        logger.error(f"Failed to get auto feedback for completion {completion_id}")
+        return None
+
+    if response.status_code == 200:
+        return response.json()
+    else:
+        response.raise_for_status()
+
+
 @click.command()
 @click.option("--task_id", help="Feedback task ID")
 @click.option("--content", help="Completion content")
 @click.option("--file", "-f", help="File containing completion content")
 @click.option("--completion_id", help="Completion ID")
 @click.option("--num_samples", default=5, help="Number of samples to use for few-shot learning")
 def auto_feedback_icl(task_id: str, content: str, file: str, completion_id: str, num_samples: int):
+    """
+    Generate feedback with existing feedback
```

Comment:
same for the doc. `Generate feedback with existing human feedback based on in context learning`